src/App.js

```diff
@@ -57,300 +59,171 @@ class App extends React.Component {
         button.key !== "AC" &&
         button.key !== ")"
       ) {
+        let result = this.state.result;
+        button = button.key;
```

Comment:
@grimxreaper Changing the meaning of button is not a safe edit. You need to rename the variable to a unique name inside the function before inlining it. If I'm seeing it correctly this may have introduced a breaking change where you set operate to "button.key" later. Please reset the commits and try again.

Reply:
reset all the commits?

Reply:
@grimxreaper Reset to just before inlining changeKeys and make sure you have introduced unique variable names when inlining. `button` needs to be renamed to something unique before inlining.